lib/vrl/tests/tests/expressions/group/precedence.vrl

```diff
@@ -1,3 +1,3 @@
 # result: 2
 
-(0 + 2) * ((2 + 3) % 2)
+(0 + 2) * (mod((2 + 3), 2))
```

Comment:
It might be best to move this test away from using modulus. The test is supposed to validate group precedence, but now anything to the right of the multiplication only has groups that have no impact on precedence at all.